config.js

```diff
@@ -23,3 +23,32 @@
 
 'use strict';
 
+var config = {};
+
+config.mqtt = {
+    host: 'localhost',
+    port: 1883,
+    defaultKey: '1234'
+};
+
+config.iota = {
+    logLevel: 'DEBUG',
+    contextBroker: {
+        host: '52.37.157.18',
+        port: '10026'
+    },
+    server: {
+        port: 4041
```

Comment:
(minor comment)
are this port and the contextBroker.host IP correct for a default config file? I mean, the port is the same as the one for the mqtt iotagent and the CB host is not localhost. An installation script could have problems with this default values when it tries to search for them to change them.